homeassistant/components/unifiprotect/entity.py

```diff
@@ -125,7 +125,7 @@ def __init__(
             self.device: ProtectAdoptableDeviceModel = device
 
         if description and not hasattr(self, "entity_description"):
-            self.entity_description = description
+            self.entity_description = description  # pragma: no cover
```

Comment:
Is this actually used in production?

Reply:
No. All of the child classes define it before passing it in. But it is another type check issue. I could assert here to make the type checker happy and then no need for the no cover.

Reply:
Well, with the changes the frenck recommended to put the typing in the class def for each child class. I might be able to start passing it into the base class now to let it define it. Let me play around with it.

Reply:
Thats much nicer 👍